src/imperator/data/deity_categories.rs

```diff
@@ -23,7 +23,6 @@ impl DbKind for DeityCategory {
     fn validate(&self, key: &Token, block: &Block, data: &Everything) {
         let mut vd = Validator::new(block, data);
 
-        data.verify_exists(Item::Localization, key);
         let loca = format!("{key}").to_uppercase();
```

Comment:
The `format` here is a no-op. You could do `let loca = key.as_str().to_uppercase()` instead.